nodePackages/default.nix

```diff
@@ -3099,6 +3099,7 @@
     module-not-found-error = callPackage ./module-not-found-error/1.0.1.nix {};
     module-not-found-error_1-0-1 = callPackage ./module-not-found-error/1.0.1.nix {};
     moment = callPackage ./moment/2.12.0.nix {};
```

Comment:
Any reason not to bump the default version of moment now?  Normally, it's pointed at the latest version.